R/pip.R

```diff
@@ -188,11 +193,24 @@ pip <- function(country         = "ALL",
     out <- censor_rows(out, lkup[["censored"]], type = "countries")
   }
 
+
   # Select columns
-  out <- out[, .SD, .SDcols = lkup$pip_cols]
+  if (additional_ind) {
+    additional_indicators(out)
+    added_names <- attr(out, "new_indicators_names")
+    names2keep  <- c(lkup$pip_cols, added_names)
+
+    # Keep relevant variables
+    out  <- out[, ..names2keep]
```

Comment:
Same comment as before about Non Standard Evaluation and performance.

Reply:
We do this type of evaluation all the time throughout our packages. I think we could leave this one like this.

Reply:
We do indeed, but I've been removing some of them specifically for that reason, as it was creating performance bottlenecks. I'm really not sure why data.table is sometime so fast, and sometimes creates performance issues... Let's leave it like this, and we can revisit if we run into performance issues.